.azure-pipelines/gpu-tests.yml

```
@@ -18,18 +18,21 @@ pr:
 
 jobs:
   - job: pytest
+    strategy:
+      matrix:
+        'PyTorch - LTS':
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
+        'PyTorch - stable':
```

nit: can we shorten this text?

```suggestion
        'LTS':
          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
        'stable':
```